example_workspace/src/hello_world_example_app/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.4.1)
+
+project(hello_ros)
+
+find_package(catkin REQUIRED COMPONENTS roscpp std_msgs rosconsole)
+
+include_directories(${catkin_INCLUDE_DIRS})
+
+# build native_app_glue as a static lib
+set(${CMAKE_C_FLAGS}, "${CMAKE_C_FLAGS}")
```

Comment:
What is the effect of this line? Seems to be invalid cmake depending on the current value of `CMAKE_C_FLAGS`.